pkg/metrics/buildinfo/buildinfo.go

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package buildinfo
```

Comment:
Can we move this file under `pkg/version/metrics.go` instead of defining a new package here? I think it fits there.

Reply:
@lambdanis that's good idea, thanks feedback